src/main/java/com/stripe/Stripe.java

```diff
@@ -206,4 +212,82 @@ public static void setAppInfo(String name, String version, String url, String pa
   public static Map<String, String> getAppInfo() {
     return appInfo;
   }
+
+  /**
+   * Send raw request to Stripe API. This is the lowest level method for interacting with the Stripe
+   * API. This method is useful for interacting with endpoints that are not covered yet in
+   * stripe-java.
+   *
+   * @param method the HTTP method
+   * @param relativeUrl the relative URL of the request, e.g. "/v1/charges"
+   * @param content the body of the request as a string
+   * @return the JSON response as a string
+   */
+  public static StripeResponse rawRequest(
+      final ApiResource.RequestMethod method, final String relativeUrl, final String content)
+      throws StripeException {
+    RawRequestOptions options = RawRequestOptions.builder().build();
+    return rawRequest(method, relativeUrl, content, options);
+  }
+
+  /**
+   * Send raw request to Stripe API. This is the lowest level method for interacting with the Stripe
+   * API. This method is useful for interacting with endpoints that are not covered yet in
+   * stripe-java.
+   *
+   * @param method the HTTP method
+   * @param relativeUrl the relative URL of the request, e.g. "/v1/charges"
+   * @param content the body of the request as a string
+   * @param options the special modifiers of the request
+   * @return the JSON response as a string
+   */
+  public static StripeResponse rawRequest(
+      final ApiResource.RequestMethod method,
+      final String relativeUrl,
+      final String content,
+      final RawRequestOptions options)
+      throws StripeException {
+    String url = ApiResource.fullUrl(Stripe.getApiBase(), options, relativeUrl);
+    StripeRequest request = StripeRequest.createWithStringContent(method, url, content, options);
+
+    Map<String, String> additionalHeaders = options.getAdditionalHeaders();
+
+    if (additionalHeaders != null) {
+      for (Map.Entry<String, String> entry : additionalHeaders.entrySet()) {
+        String key = entry.getKey();
+        String value = entry.getValue();
+        request = request.withAdditionalHeader(key, value);
+      }
+    }
+
+    if (options.getApiMode() == ApiMode.PREVIEW) {
+      request = request.withAdditionalHeader("Stripe-Version", ApiVersion.PREVIEW_CURRENT);
+    }
+
+    StripeResponseStream responseStream = new HttpURLConnectionClient().requestStream(request);
```

Comment:
Just noticed this. In all other cases of request sending we go through the `StripeResponseGetter`. It allows users to replace the HttpClient used and change it's settings. By going directly through the `HttpURLConnectionClient` here we are creating a separate way of making requests.

Reply:
Very good point, just fixed.